modules/registry-securing-manually.adoc

```diff
@@ -13,11 +13,11 @@ sections for steps on how to manually do so.
 Manually secure the registry to serve traffic via TLS:
 
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
-. Deploy the registry using the `oc adm registry` command as a
+. Deploy the registry using the `oc registry` command as a
 user with cluster administrator privileges, for example:
 +
 ----
-$ oc adm registry --config=/etc/origin/master/admin.kubeconfig \//<1>
+$ oc registry --config=/etc/origin/master/admin.kubeconfig \//<1>
```

Comment:
I don't think command 'oc registry' can deploy registry manually. You can see its function is as below:
Advanced commands:
  info        Print info about the integrated registry
  login       Login to the integrated registry